packages/gatsby-image/gatsby-browser.js

```diff
@@ -0,0 +1,14 @@
+"use strict"
```

Comment:
Whoops - forgot to comment on this last time. Should this file be in src instead? Or not committed to version control?

Reply:
I'm not sure how it works, actually! This is my first time working on a plugin...

It contains the code that imports the polyfill, so would it still work without committing it? I'm happy to move it or change whatever you recommend. 👍